src/aggregate/expr/expression.c

```diff
@@ -169,6 +169,7 @@ static int evalPredicate(ExprEval *eval, const RSPredicate *pred, RSValue *resul
 
 static int evalProperty(ExprEval *eval, const RSLookupExpr *e, RSValue *res) {
   if (!e->lookupObj) {
```

Comment:
it was a possibility at one point... though I think we tried to make sure this doesn't happen anymore. Please change